spice-labs.sh

```diff
@@ -65,7 +68,7 @@ fail_if_unset() {
 }
 
 check_binaries() {
-  for bin in /opt/docker/bin/goatrodeo /usr/bin/ginger ; do
+  for bin in "$GOAT_RODEO_PATH" "$GINGER_PATH"; do
```

Comment:
Any reason they're in specific but overridable places, rather than in the PATH? This can be tested with `which`